maths/sum_of_harmonic_series.py

```diff
@@ -0,0 +1,37 @@
+def sum_of_hp(first_term, common_difference, no_of_terms):
+
+    """
+
+    Find the sum of n terms in an harmonic progression.
+    common_diff is the common difference of Arithmetic
+    Progression by which the given Harmonic Progression is linked.
+
+
+    """
+
+    l1 = [1 / first_term]
```

Comment:
We require self-documenting variable names.
```suggestion
    arithmetic_progression = [1 / first_term]
```